RecoMET/METFilters/python/BadPFMuonFilter_Dz_cfi.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+BadPFMuonFilter = cms.EDFilter(
+    "BadParticleFilter",
+    PFCandidates  = cms.InputTag("particleFlow"),   # Collection to test
+    muons  = cms.InputTag("muons"),   # Collection to test 
+    vtx = cms.InputTag("offlinePrimaryVertices"), 
+    taggingMode   = cms.bool(False),
+    filterType  =cms.string("BadPFMuonDz"),
+    maxDR         = cms.double(0.001),              # Maximum DR between reco::muon->innerTrack and pfCandidate 
+    minPtDiffRel = cms.double(0.0),               # lower threshold on difference between pt of reco::muon->innerTrack and pfCandidate
+    algo          = cms.int32(14),
+    minMuonTrackRelErr    = cms.double(2.0),               # threshold on best track RelptError
+    innerTrackRelErr = cms.double(1.0),            # threshold on innerTrack relPtErr
+    minDxyBestTrack = cms.double(-1.0),
+    minDzBestTrack = cms.double(-1.0),
```

Comment:
why is minDzBestTrack not set to some default value > 0 in this config implementing the filter including a dz cut?